src/components/ProofFooter.vue

```diff
@@ -13,9 +13,9 @@
       </span>
       
     </v-chip>
-    <PriceCountChip :count="proof.price_count" :withLabel="true"></PriceCountChip>
+    <PriceCountChip v-if="!reducedProofFooter" :count="proof.price_count" :withLabel="true"></PriceCountChip>
```

Comment:
- why hide the price count ? seems a useful info to display
- and I'd rename `reducedProofFooter` to `hideProofDelete` (mecanism similar to other components)

Reply:
the first `v-chip` for proof type is also hidden, so it would be named `hideProofTypeDelete`, but I suppose we should decide first what chip to keep before naming this boolean

Reply:
why hide the proof type then ? :upside_down_face:

Reply:
Taking a lot of space, and info not needed to in this case

Reply:
I disagree, I wouldn't want to add GDPR proof here.
But in any case this is not important, you can keep it as is, it was just to simplify the logic